interfaces/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@binaris/shift-interfaces",
-  "version": "0.3.0",
+  "version": "0.4.0",
```

Comment:
Should all the dependencies (`db`, `subscriptions`) be updated to use this version?

Reply:
Not yet.  `db` is going away but not yet.  `subscriptions` is even further away, for now.